torch/distributed/_composable/fully_shard.py

```diff
@@ -81,4 +85,10 @@ def fully_shard(
     _register_pre_forward_hooks(state, modules)
     _register_post_forward_hooks(state, modules)
     _register_root_pre_forward_hook(state, module)  # prepend last
+    for submodule in module.modules():
+        if (
+            state._ignored_modules is not None
```

Comment:
nit: Do we need this `is not None` check? IIUC, `state._ignored_modules` should not be `None`.